aiohttp/connector.py

```diff
@@ -347,8 +354,12 @@ def closed(self):
         """
         return self._closed
 
-    async def connect(self, req):
+    async def connect(self, req, trace_context=None):
```

Comment:
Let's pin `trace_context` to mandatory params set.
I'm ok with breaking backward compatibility in connectors API.

Reply:
The same for other places with `trace_context=None` in signatures.

Reply:
Oks.